app/assets/javascripts/backbone/views/timeline_batch_actions_view.coffee

```diff
@@ -23,6 +24,23 @@ class MS.Views.TimelineBatchActionsView extends Backbone.View
 
     $form.submit()
 
+  # Check the selected steps for any steps with precedents
+  # If the step has a precedent step, show a message
+  checkForUnselectedPrecedents: (stepIds) ->
+    @$('#steps-notice').hide()
+
+    dependents = @$(".select-step[data-id][data-precedent-id]:checked")
+    stepIds = @$('.step-ids').val()
```

Comment:
There are a few odd things going on with `stepIds` here it seems. First of all, it's passed in as an arg and then overwritten. That's easily enough fixed.

But also it seems like in TimelineSelectStepsView, we're assigning an array of IDs to the value of the `.step-ids` input. If you try this out (I had to to make sure!) you'll see that this sets the input's value to a string of comma separated IDs like "124,224,824" etc.

Then below you're doing `includes`, which is a string function that checks if a substring is present in the string. This is a problem because `"124,224,824".includes(24)` will be true, which is not what you want.

Note that `disabled = stepIds.length < 1` works because it's checking that the string of IDs is empty.

To fix this you could split the ID string by comma and test against that. I would add a method `selectedStepIds` that does this (`(@$('.step-ids').val() || '').split(',')`) and call it from both places.

Then I don't think you need to pass `stepIds` to `checkForUnselected...`.